src/KeyVault/KeyVault/help/Update-AzKeyVaultSecret.md

```diff
@@ -63,7 +63,9 @@ ContosoVault, using the stored variables.
 
 ### Example 2: Delete the tags and content type for a secret
 ```powershell
-Update-AzKeyVaultSecret -VaultName 'ContosoVault' -Name 'HR' -Version '9EEA45C6EE50490B9C3176A80AC1A0DF' -ContentType '' -Tag -@{}
+$Tags = @{ 'Severity' = 'medium'; 'HR' = 'true'}
+$ContentType= 'xml'
+Update-AzKeyVaultSecret -VaultName 'ContosoVault' -Name 'HR' -Version '9EEA45C6EE50490B9C3176A80AC1A0DF' -ContentType $ContentType -Tag $Tags
```

Comment:
-Tag @{} (delete the dash before @)

No other errors here. The purpose of this command is to delete, so blank value is assigned.